Assets/Controllers/GameManager.cs

```diff
@@ -153,4 +179,36 @@ public Vector3 GetPawnPosition(Pawn pawn)
         Vector3 pawnPosition = pawnKeeper.GetPawnGOFromPawn(pawn).transform.position;
         return pawnPosition;
     }
+
+    public void EndTurn()
+    {
+        int playerCount = players.Count;
+
+        int playerIndex = players.FindIndex( x => x == activePlayer);
+
+        if(playerIndex + 1 == playerCount)
+        {
+            playerIndex = 0;
+        } else
+        {
+            playerIndex++;
+        }
+        ClearSelectedPawn();
+        activePlayer = players[playerIndex];
+        activePlayer.StartTurn();
+        GetFirstPawnForActivePlayerPositionAndFocusCamera();
+    }
+
+    public void GetFirstPawnForActivePlayerPositionAndFocusCamera()
+    {
+        Pawn pawn = GetFirstPawnForActivePlayer();
+        if (pawn != null)
+        {
+            Vector3 pawnPosition = GetPawnPosition(pawn);
+            Camera.main.GetComponent<CameraMotionHandler>().MoveToPosition(pawnPosition);
+        } else
+        {
+            //probably make the End Turn button flash yellow or something
```

Comment:
I'd suggest giving it a default basic behavior until you are ready to extend it.